kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/RemoteFormModellerFormProvider.java

```diff
@@ -333,6 +337,81 @@ protected String filterXML(String document, String lang, String deploymentId, Ma
         return document;
     }
 
+    protected String attachSubForms(String document, String deploymentId) {
```

Comment:
This seems like a reduced copy/paste of filterXML method. It would be nice to refactor it to reduce code duplication